src/serializers/GltfSerializer.cpp

```diff
@@ -121,17 +121,28 @@ const uint32_t component_type<int>::value = CT_UNSIGNED_INT;
 template <>
 const uint32_t component_type<float>::value = CT_FLOAT;
 
+static int bufferViewId = 0;
+
 template <size_t N, typename It>
 size_t write_accessor(json& j, std::ofstream& ofs, It begin, It end) {
 	auto num = std::distance(begin, end) / N;
 
 	json accessor = json::object();
 
-	accessor["bufferView"] = N == 1 ? 0 : 1;
-	accessor["byteOffset"] = (size_t)ofs.tellp();
+	accessor["bufferView"] = bufferViewId;
+	accessor["byteOffset"] = 0;
 	accessor["componentType"] = component_type<typename It::value_type>::value;
 	accessor["count"] = num;
 
+	if (N == 1) {
+		j["bufferViews"].push_back({ {"buffer", 0}, {"byteOffset", (size_t)ofs.tellp()}, { "byteLength", num *  4}, {"target", 34963} });
+	}
+	else {
```

Comment:
```suggestion
	} else {
```

We don't have a proper C++ style guide yet. But this is more inline.

Reply:
ok